nameko/amqp.py

```diff
@@ -43,7 +43,7 @@ class TestTransport(Transport):
 
 def verify_amqp_uri(amqp_uri):
     connection = Connection(amqp_uri)
-    if connection.transport_cls != 'amqp':
+    if connection.transport_cls not in ('amqp', 'pyamqp'):
```

Comment:
What do you mean?

I guess you could call it with `librabbitmq://`, but that would break in all kinds of other ways because it doesn't work with eventlet. What was the original purpose of this check?